### PR TITLE
Erg #76 column horizontal line

### PR DESCRIPTION
**Pull request by derpdead**

### CODE REVIEW

File: components/Grid/GridColumn.vue [hunk 1]
```diff
@@ -63,8 +109,312 @@ export default {
...
+            this.removeEventListenersForResizeState();
+            this.isResizing = false;
+        },
+        initMousePosition({ clientX }) {
+            this.startX = clientX;
+        },
+        initElementWidth() {
+            const {
+                width: elementWidth,
+            } = this.$el.getBoundingClientRect();
```
Comment (bleto) [inline components/Grid/GridColumn.vue]: you use `this.$el.getBoundingClientRect();` a couple of times. Maybe you can put it in the method.

File: components/Grid/GridColumn.vue [hunk 2]
```diff
@@ -63,8 +109,312 @@ export default {
...
+
+            this.startWidth = parseInt(elementWidth, 10);
+        },
+        initElementStyleForResizeState() {
```
Comment (bleto) [inline components/Grid/GridColumn.vue]: you can use `updateElementWidth()` method with `this.startWidth` parameter.

File: components/Grid/GridColumn.vue [hunk 3]
```diff
@@ -63,8 +109,312 @@ export default {
...
+            const neighbourIndex = this.index === 0 ? this.index : this.index - 1;
+
+            if (!isMouseAboveColumnHeader
+                || this.isPinnedLeft
+                || this.isPinnedRight
+                || this.isExtenderColumn
+                || this.isResizing) {
+                event.preventDefault();
+
+                return false;
```
[the remark this answers is not in the document]
Comment (derpdead) [reply components/Grid/GridColumn.vue]: I do, otherwise code bellow will be executed

File: components/Grid/GridColumn.vue (hunk 1 above)
Comment (derpdead) [reply components/Grid/GridColumn.vue]: Why would I ? It is already method call

**Pull request merged**